.circleci/config.yml

```diff
@@ -83,13 +83,13 @@ executors:
   # due to garbage collection timeout. This should be reassessed after upgrading to Junit 5. See RW-6460.
   api-unit-test-executor:
     <<: *workbench-executor
-    resource_class: medium+
+    resource_class: large
```

Comment:
Were tests not working with medium+ with the package upgrade?

Reply:
That's right